tox.ini

```diff
@@ -1,10 +1,23 @@
 [tox]
-envlist = py26, py27, py32, py33, py34, pypy
+envlist = py26, py27, py32, py33, py34, pypy, gae
 
 [testenv]
+passenv = PYTHONPATH TRAVIS*
 deps= -r{toxinidir}/dev-requirements.txt
 commands=
-   nosetests \
-   []
+    nosetests []
 setenv =
     PYTHONWARNINGS=always::DeprecationWarning
+
+[testenv:gae]
+basepython = python2.7
+deps= 
+    -r{toxinidir}/dev-requirements.txt
+    nosegae
```

Comment:
Could we move this to its own requirements file with a version pin?
Including all deps.